apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx

```diff
@@ -53,10 +60,9 @@ function BackToInternalDomain({
 
   useEffect(() => {
     (async () => {
-      const lastScreen = (await AsyncStorage.getItem("last-screen")) || "";
-      setButtonText(lastScreen === "compare_providers" ? t("common.quote") : manifest.name);
+      setButtonText(btnText);
```

Comment:
Why are we setting a state with `btnText` here?
Can't we just use the value that is coming from the config?